tools/attrd_updater.c

```diff
@@ -455,16 +456,75 @@ send_attrd_query(pcmk__output_t *out, const char *attr_name, const char *attr_no
 }
 
 static int
-do_update(char command, const char *attr_node, const char *attr_name,
-          const char *attr_value, const char *attr_section,
-          const char *attr_set, const char *attr_dampen, int attr_options)
+send_attrd_update(char command, const char *attr_node, const char *attr_name,
+                  const char *attr_value, const char *attr_set,
+                  const char *attr_dampen, int attr_options)
```

Comment:
I was pretty sloppy with section handling, and in fact removed it from these new functions on the basis of it being marked as "Not yet implemented" in the help output for attrd_updater.

Reply:
Yep, but implementing it will be a project down the line :)

In any case, we don't need to pass a separate argument for it, we can use a flag when the time comes.

Reply:
I guess I'm still not quite following here.  Previously, section was a string and we at least cared about "reboot" and "forever", and mapped those to "status" and "nodes".  Are those two the only valid settings for section, or could it be other things?  The code for attrd_updater makes it seem like it could be other things.

Reply:
Yep all that really matters is transient or permanent. Status/reboot = transient, nodes/forever = permanent.

Reply:
Okay, now it's all coming together.